packages/nodes-base/nodes/Code/Code.node.ts

```diff
@@ -66,7 +66,7 @@ export class Code implements INodeType {
 	};
 
 	async execute(this: IExecuteFunctions) {
```

Comment:
can we add a test here to avoid this bug happening again?

Reply:
done